### ISSUE TITLE
Language

### ISSUE THREAD

**Issue by nicktgbrown** [acgd-webdev-3/accessibility-fixer-upper#1]
Having issues with the <html lang="en-ca"> tag, unsure how to fix it.

<img width="506" alt="screen shot 2017-12-01 at 8 02 06 pm" src="https://user-images.githubusercontent.com/25106751/33509798-d15a1edc-d6d2-11e7-9659-61859c3e30fc.png">

@hellachella

**Comment by thomasjbradley** [comment]
Yeah, that error is super confusing. I couldn't figure out how to make it better. Do you have all the ARIA landmark roles? `main`, `banner`, `contentinfo`, `navigation`, etc.

p.s. In the future, please create the issue attached to your code instead of mine.
